Shared/Samples/Trace utility network/TraceUtilityNetworkView.Model.swift

```diff
@@ -0,0 +1,353 @@
+        }
+        
+        /// Performs important tasks including adding credentials, loading and adding operational layers.
+        func setup() async {
+            do {
+                try await ArcGISEnvironment.authenticationManager.arcGISCredentialStore.add(.publicSample)
+                try await network?.load()
+            } catch {
+                await updateUserHint(withMessage: "An error occurred while loading the network.")
+                return
+            }
+            featureLayerURLs.forEach { url in
+                let table = ServiceFeatureTable(url: url)
+                let layer = FeatureLayer(featureTable: table)
+                if table.serviceLayerID == 3 {
+                    layer.renderer = UniqueValueRenderer(
+                        fieldNames: ["ASSETGROUP"],
+                        uniqueValues: [.lowVoltage, .mediumVoltage],
+                        defaultSymbol: SimpleLineSymbol()
+                    )
+                }
+                map.addOperationalLayer(layer)
+            }
+        }
+        
+        /// Runs a trace with the pending trace configuration and selects features in the map that
+        /// correspond to the element results.
+        ///
+        /// - Note: Elements are grouped by network source prior to selection so that all selections
+        /// per operational layer can be made at once.
+        func trace() async throws {
+            guard let pendingTraceParameters = pendingTraceParameters else { return }
+            let traceResults = try await network?.trace(using: pendingTraceParameters)
+                .filter { $0 is UtilityElementTraceResult }
+            for result in traceResults as? [UtilityElementTraceResult] ?? [] {
+                let groups = Dictionary(grouping: result.elements) { $0.networkSource.name }
+                for (networkName, elements) in groups {
+                    guard let layer = map.operationalLayers.first(
+                        where: { ($0 as? FeatureLayer)?.featureTable?.tableName == networkName }
+                    ) as? FeatureLayer else { continue }
+                    let features = try await network?.features(for: elements) ?? []
+                    layer.selectFeatures(features)
+                }
+            }
+        }
+        
+        /// Updates the textual user hint.
+        ///
+        /// If no message is provided a default hint is used.
+        /// - Parameter message: The message to display to the user.
+        @MainActor func updateUserHint(withMessage message: String? = nil) {
+            if let message {
+                hint = message
+            } else {
+                switch tracingActivity {
+                case .none:
+                    hint = ""
+                case .settingPoints(let pointType):
+                    switch pointType {
+                    case .start:
+                        hint = "Tap on the map to add a Start Location."
```

Comment:
```suggestion
                        hint = "Tap on the map to add a start location."
```

Reply:
I was matching the [capitalization of ObjC](https://github.com/Esri/arcgis-runtime-samples-ios/blob/b2694f9e5a97cd81ea042c1078708215ab4f3f48/arcgis-ios-sdk-samples/Utility%20network/Trace%20utility%20network/TraceUtilityNetworkViewController.swift#L351-L358) here but I do like the lowercase forms better. I'll take the suggestions and leave it for @yo1995 to comment on if he likes.

Reply:
No preference 🙈